### PR TITLE
i18n(es): Translated `middleware-no-data-or-next-called.mdx`

### PR DESCRIPTION
**Pull request by Waxer59**
#### What kind of changes does this PR include?

- New or updated content
- Translated content

#### Description

- What does this PR change? Give us a brief description.
Translated `middleware-no-data-or-next-called.mdx` to spanish

### CODE REVIEW

File: src/content/docs/es/reference/errors/middleware-no-data-or-next-called.mdx [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+> **MiddlewareNoDataOrNextCalled**: El middleware debe devolver un objeto `Response` o llamar a la función `next`. (E03031)
+
+## ¿Qué salió mal?
+Se produce un error cuando el middleware no devuelve ningún dato ni llama a la función `next`.
+
+Por ejemplo:
+```ts
+import {defineMiddleware} from "astro/middleware";
+export const onRequest = defineMiddleware((context, _) => {
+	// doesn't return anything or call `next`
```
Comment (dreyfus92) [inline src/content/docs/es/reference/errors/middleware-no-data-or-next-called.mdx:16]:
```suggestion
	// no regresa nada o llama `next
```

**Review by dreyfus92 — CHANGES_REQUESTED** [review]
Buen trabajo, te dejo una sugerencia @Waxer59 😁

**Comment by Waxer59** [comment]
Gracias por la revision 🤠

**Review by Yan-Thomas — APPROVED** [review]
LGTM! Thanks for the awesome work @Waxer59 & @dreyfus92 💜

**Pull request merged**